.github/workflows/ci.yml

```diff
@@ -168,9 +168,15 @@ jobs:
     - name: Update pip, wheel, setuptools, build, twine
       run: |
         python -m pip install -U pip wheel setuptools build twine
+    - name: Test a workaround for #6600
+      if: startsWith(matrix.pyver, '3.11')
+      run: python -m pip install --upgrade cchardet
```

Comment:
This fails on Python 3.11-beta.4

Reply:
See #6819.
Feel free to add something to remove it from installation in 3.11 for now.